src/apps/explorer/components/common/Search/index.tsx

```diff
@@ -27,7 +27,7 @@ export const Search: React.FC<React.HTMLAttributes<HTMLDivElement>> = (props) =>
         name="query"
         value={query}
         onChange={(e): void => setQuery(e.target.value.trim())}
-        placeholder="Search by Order ID / Address"
+        placeholder="Search by Order ID / ETH Address / ENS Address"
```

Comment:
There's one more spot where this should be updated https://github.com/gnosis/gp-ui/blob/2b2193a9f46310cb8234b3b151cab14f9c4b5a81/src/apps/explorer/pages/Home/index.tsx#L30